lib/maths/CPeriodicityHypothesisTests.cc

```diff
@@ -2143,13 +2140,11 @@ bool CPeriodicityHypothesisTests::testVariance(const TTimeTimePr2Vec& window,
     LOG_TRACE(<< "  autocorrelation          = " << R);
     LOG_TRACE(<< "  autocorrelationThreshold = " << stats.s_AutocorrelationThreshold);
 
-    TSizeVec repeats{calculateRepeats(window, period_, m_BucketLength, buckets)};
-    meanRepeats = CBasicStatistics::mean(
-        std::accumulate(repeats.begin(), repeats.end(), TMeanAccumulator{},
-                        [](TMeanAccumulator mean, std::size_t repeat) {
-                            mean.add(static_cast<double>(repeat));
-                            return mean;
-                        }));
+    meanRepeats = [&] {
```

Comment:
ditto

Reply:
Done in [5d3c3a3](https://github.com/elastic/ml-cpp/pull/721/commits/5d3c3a3b79319afb4949e5ad10666d9adf1036d3).